webclient/components/1time-Mrow-mField/diagnosis/edit-layer/add-ct.vue

```diff
@@ -0,0 +1,298 @@
+<!-- Reference implementation -->
```

Comment:
Why is there a seperate add component. The design of SS where the same ct can do add/remove seems to be better. Look at how the search at top of SS ct can be used to filter and narrow down.

Reply:
Sir, I made the add component flown by the Remainder component.
okay, now I will change the diagnosis component flowed by the SS component.